Dockerfile

```diff
@@ -0,0 +1,18 @@
+
+FROM ubuntu:20.04
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt update && \
+    apt install curl python3.9 python3.9-dev python3.9-distutils git g++ ffmpeg libsm6 libxext6 libgl1-mesa-glx -y && \
+    curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && \
+    python3.9 get-pip.py && \
```

Comment:
Do you think we should symlink `python3.9` as `python` to make it convinient for users. Something like `ln -sf /usr/bin/python3.9 /usr/bin/python`

Reply:
Python isn't directly used by the user, so I don't think it's that big of an issue as of yet

Reply:
for more user's conveniences, how about get the python version through the `build-arg`? if it set to any supported python version as a default (e.g., python 3.10 - it's the latest version of the python which is supported by this project), most of users who does not care about the python version can use it without setting `build-arg` but someone who intended to use different version of python, they can set it via `build-arg` to build the docker image.